src-electron/generator/generation-engine.js

```diff
@@ -527,16 +538,16 @@ async function loadSingleTemplate(db, genTemplatesJson) {
     return context
   }
   context.path = file
-  await dbApi.dbBeginTransaction(db)
+  if (!isTransactionAlreadyExisting) await dbApi.dbBeginTransaction(db)
```

Comment:
What happens if transaction is already active? Don't we have to queue a new transaction that could have come?